astronomer/providers/apache/spark/example_dags/example_spark_dag.py

```diff
@@ -0,0 +1,37 @@
+"""
+Example Airflow DAG to submit Apache Spark applications using
+`SparkSubmitOperator`.
+"""
+from datetime import datetime
+
+from airflow import DAG
+from airflow.operators.dummy import DummyOperator
+
+from astronomer.providers.apache.spark.operators.spark_sumbit import (
+    SparkSubmitOperatorAsync,
+)
+
+with DAG(
+    dag_id="example_spark_operator",
+    schedule_interval=None,
+    start_date=datetime(2021, 1, 1),
+    catchup=False,
+    tags=["example", "async", "spark"],
+) as dag:
+
+    # [START howto_Dummy_operator_start]
+    start = DummyOperator(task_id="start")
+    # [END howto_Dummy_operator_start]
+
+    # [START howto_operator_spark_submit_async]
+    submit_job_sync = SparkSubmitOperatorAsync(
```

Comment:
I think this operator has optional  `conn_id` param that we should take from env variable

Reply:
Added